ExecuteAdHocBoosterUpdate Error when starting the application

When starting the application I'm getting this error:

```
client
Starting snowdrop-automation-client (/home/nvm/snowdrop-automation-client)...
> @snowdrop/snowdrop-automation-client@0.1.0 start /home/nvm/snowdrop-automation-client
> atm-start

2020-07-28T15:04:00.745Z [m:13] [info ] Starting Atomist automation client @snowdrop/snowdrop-automation-client:0.1.0
2020-07-28T15:04:01.791Z [m:13] [error] Registration failed with code '400': '"Duplicate commands found with name(s): [ExecuteAdHocBoosterUpdate]"'
2020-07-28T15:04:01.795Z [m:13] [info ] Shutdown initiated, calling shutdown hooks
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @snowdrop/snowdrop-automation-client@0.1.0 start: `atm-start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @snowdrop/snowdrop-automation-client@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/nvm/.npm/_logs/2020-07-28T15_04_01_836Z-debug.log
```

--- a/lib/commands/BoosterPropertyUpdate.ts
+++ b/lib/commands/BoosterPropertyUpdate.ts
@@ -20,7 +20,7 @@ import {FixedBranchDefaultRepoRefResolver} from "../support/repo/FixedBranchDefa
 import {setProperty} from "../support/utils/pom";
 
 @CommandHandler("Update a Maven property of the boosters", "update boosters maven property")
-export class ExecuteAdHocBoosterUpdate implements HandleCommand {
+export class BoosterPropertyUpdate implements HandleCommand {
 
   @Parameter({
     displayName: "Maven property name",

